### PR TITLE
Added navigation

### PR DESCRIPTION
**Pull request by Firanus**
Added the React Navigation and implemented two screens that the user can flip between.

### CODE REVIEW

File: __tests__/App.test.js [hunk 1]
```diff
@@ -5,5 +5,5 @@ import App from '../App';
 
 it('renders without crashing', () => {
   const rendered = renderer.create(<App />).toJSON();
```
Comment (jaherulalom) [inline __tests__/App.test.js]: Is there a reason you are adding .toJSON(), what is the purpose of it in this test case?
I don't think it's necessary?

File: __tests__/screens/HomeScreen.test.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import HomeScreen from '../../src/screens/HomeScreen';
+
+it('renders the home screen correctly', () => {
+  const navigation = { navigate: jest.fn() };
+  const rendered = renderer.create(<HomeScreen navigation={ navigation }/>).toJSON();
+  expect(rendered).toMatchSnapshot();
+});
```
Comment (jaherulalom) [inline __tests__/screens/HomeScreen.test.js]: Does the .toJSON() have something to do with the use of Snapshot?

File: __tests__/App.test.js (hunk 1 above)
Comment (Firanus) [reply __tests__/App.test.js]: We need it for the snapshot to be registered in JSON rather a JS object. :)

File: __tests__/screens/HomeScreen.test.js (hunk 2 above)
Comment (Firanus) [reply __tests__/screens/HomeScreen.test.js]: Ditto the other comment. We need it for the snapshot :)

**Pull request merged**